src/swarm/neo/connection/ConnectionBase.d

```diff
@@ -1034,6 +1048,23 @@ abstract class ConnectionBase: ISelectClient
 
     /***************************************************************************
 
+        Requests to send outgoing data buffered by the OS for the socket
+        immediately. By default outgoing data are buffered to the maximum
+        payload of a TCP frame using the Linux TCP_CORK socket option.
+
+        This method is intended to be called right after `getPayloadForSending`
```

Comment:
Ok, maybe it's something we can add to the all-in-one method, when that's merged.